lib/active_attr/typecasting/boolean_typecaster.rb

```diff
@@ -42,7 +42,7 @@ def call(value)
         case value
         when *FALSE_VALUES then false
         when *NIL_VALUES then nil
-        when Numeric, /\A[-+]?(0+\.?0*|0*\.?0+)\z/ then !value.to_f.zero?
+        when Numeric, /\A[-+]?(0++\.?0*|0*\.?0+)\z/ then !value.to_f.zero?
```

Comment:
In your original issue (#184) you also changed `|0*` to `|0*+`. Is that no longer desirable?

Reply:
Sorry, there's been a correction.
I fixed it as per the issue.